src/Extensions/Logging/InitStaticLogger.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the uls logger is set
```

Comment:
nit:We don't use ULS term in opensource since it could be confusing, maybe here we could replace it with something like "before proper ILogger from DI"